### ISSUE TITLE
No way to write an additional note using mt_text_more

### ISSUE THREAD

**Issue by tousei-seiso** [OpenLiveWriter/OpenLiveWriter#783]
metaWeblog.newPost, editPost has optional parameter mt_text_more for the purpose of writing additional notes.

I'd been using a XML-RPC client 'BlogWrite' and it had a interface to write an additional notes but it turned not to work well with Windows browser component upgraded to IE9+ and the developer disappeared.

Since then I started to write a post with Open Live Writer but Open Live Writer doesn't have a way to write an additional notes. I looked into the source codes and found that Open Live Writer is operating  mt_text_more internally but has not any user interfaces to input it.

For example, if you make the Open Live Writer like the image below, we will be able to write an additional notes naturally(Though I have no idea to preview body and addition simultaneously with blog theme) : 
![2018-12-22](https://user-images.githubusercontent.com/45998954/50372305-bb68e280-060f-11e9-954e-e119fa2f34cc.png)

Is there anyone who have any suggestions for implementing this feature?

**Comment by robertdknight** [comment]
What are the additional notes for?  What do they do with regards to the final blog entry?

**Comment by tousei-seiso** [comment]
>What are the additional notes for?  What do they do with regards to the final blog entry?
Normally 'Extended' is used when you write an article, once it is published, and you want to add additional postings.

Many blogs support this feature. It is often called '**Extended**' text.
For example, Movable Type and Nucleus CMS has both 'Body' and 'Extended' fields to write a post.
So, **Movable Type XML-RPC API** offers **mt_text_more** parameter for this purpose.

But unfortunately there is no way to write 'Extended' text by Open Live Writer. I mean I'd like to add this feature to Open Live Writer.(Please see the pic in my first post. 'Addition' tab='Extended'.)

**Comment by tousei-seiso** [comment]
After my previous post, I found that Open Live Writer already has the extended documents feature.

When uploading a post to the server, by inserting  tag to split body content and extended document, Open Live Writer sets the document before the tag to description and sets the document after the tag to mt_text_more.

When downloading a existing post from the server, Open Live Writer sets the description to body document and add the  tag and the value of mt_text_more parameter after the document.

I confirmed that.
